### PR TITLE
Endpoints project

### PR DESCRIPTION
**Pull request by peterghrong**
# Summary

Initial pr just to get it started
	
## Test Plan

N/A

## Related Issues

Project Endpoint

Closes #

### CODE REVIEW

**Comment by celinehuang** [comment]
you can request review from me when you're done, and there is also a option to make this a draft PR – top right hand sidebar under reviewers

**Comment by peterghrong** [comment]
Good Morning, I have now implemented a getter as you have mentioned, "get_all_project_types" will return a jsonified list of all project types from the database. The second function "get_project_of_type" will call the filter on the database and return all projects of that specified type.

File: backend/app/project/routes.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+@project.route("/project/types", methods=["GET"])
+def get_all_project_types():
+    types = []
+    for type in Project.query.distinct(Project.type):
+        types.append(Project.type)
+    return jsonify(types)
+
+
+# return projects with the specified type
+@project.route("/project/<type>", method=["GET"])
```
Comment (celinehuang) [inline backend/app/project/routes.py:16]: does this work with `method` ?

**Review by celinehuang — COMMENTED** [review]
Could you write tests to make sure that this works? Here are some examples of endpoint tests

**Comment by celinehuang** [comment]
You can also download [Postman](https://www.postman.com/) and test your endpoints there

**Comment by peterghrong** [comment]
Ill try and test it with POSTMAN

File: backend/app/project/routes.py (hunk 1 above)
Comment (peterghrong) [reply backend/app/project/routes.py:16]: Can't believe I missed this

**Comment by peterghrong** [comment]
I will open an official pr for this issue

**Pull request closed without merging**